packages/cdktf-cli/bin/cmds/helper/synth-stack.ts

```diff
@@ -117,14 +118,21 @@ Command output on stderr:
 
   public static async synthTelemetry(
     command: string,
-    totalTime: number
+    totalTime: number,
+    stacks: SynthesizedStack[]
   ): Promise<void> {
     const reportParams: ReportParams = {
       command: "synth",
       product: "cdktf",
       version: versionNumber(),
       dateTime: new Date(),
-      payload: { command: command, totalTime: totalTime },
+      payload: {
+        command: command,
```

Comment:
I guess this isn't new, but could it potentially be exposing information that users may not want to share?

Reply:
I think we need to revise what to track in general, afaik @danieldreier is making a plan here. I'd say we leave it as is for now and improve in the next iteration. #826 is the issue tracking the entire effort, we'll update this with the concrete scope also in regards of where we might be tracking secrets or similar things by accident.